colony/commands/base.py

```diff
@@ -25,22 +25,18 @@ def __init__(self, command_args: list, connection: ColonyConnection = None):
 
         self.args = docopt(self.__doc__, argv=command_args)
 
-    def execute(self):
+    def execute(self) -> bool:
         """Finds a subcommand passed to with command in
         object actions table and executes mapped method"""
 
         args = self.args
 
         actions_table = self.get_actions_table()
-
         for action in actions_table:
-            if args.get(action, False):
-                # call action
-                actions_table[action]()
-                break
-
-        # if subcommand was specified without args (actions), just show usage
-        raise DocoptExit
+            if action in args:
+                return actions_table[action]()
+        else:
+            raise DocoptExit()
```

Comment:
I dont think that this code works. Did you test it?
I suggest to revert to the way it was done previously and I will refactor it later